README.md

```diff
@@ -32,10 +36,39 @@ If you want to build the installer run
 
 `ant opendcs`
 
+# General Development
+
+To verify everything can work on your system run the following:
+
+```
+ant test
+# NOTE: this will flash a few interfaces onto your display, let the task finish or the tests get stuck. 
+# However, you can just run through the GUIs to finish the tests. Though be aware if you don't follow the 
+# programmed script the task may return failure.
+ant gui-test -Dno.docs=true
+ant integration-test -Dno.docs=true -Dopendcs.test.engine=OpenDCS-XML
+# and if you have docker
+ant integration-test -Dno.docs=true -Dopendcs.test.engine=OpenDCS-Postgres
+```
+
+This will run all of the various tests and let you know you have everything setup such that you can start development.
+
+For all test tasks you can add `-DdebugPort=<a port number>` and the JVMs started will wait for a debug connection.
+Beware that gui-test and integration-test depend on test running, so you will have to attach the remote debugger twice.
+This is a current limitation of the ant build.
+
+See https://opendcs-env.readthedocs.io/en/latest/dev-docs.html for guidance on some of the newer components.
+
 # IDE integration
 
-For the purposes of setting up the project VS Code or Eclipse you can run the following task:
+While the OpenDCS maintainers don't have time to maintain files for other IDEs, we want to make
+it as easy as possible for anyone to help and will welcome baseline configuration for others.
+If you wish to help in this way please created additional targets and required files to generate them.
+
+## Visual Studio Code or Eclipse
+
+To create the appropriate project files for VS Code or Eclipse you can run the following task:
 
 `ant eclipse-ide-files`
 
-This will create the appropriate .project and .classpath files for code completion and following.
+This will create the appropriate .project and .classpath files for intellisense to behave correctly.
```

Comment:
consider adding this:

## IntelliJ 

Some notes on using IntelliJ:  https://github.com/opendcs/opendcs/wiki/Coding-OpenDCS-with-IntelliJ